pants_plugins/sendwave/pants_docker/BUILD

```diff
@@ -1,19 +1,24 @@
-python_library(
+python_sources(
```

Comment:
This name changed in the 2.9.x line: https://www.pantsbuild.org/docs/reference-python_sources